codice/ddf
# [2.11.x] DDF-3466 Added an interface to get system high attributes

#### What does this PR do?
This PR adds an interface and impl to get system high attribute values.
#### Who is reviewing it? 
@peterhuffer @AzGoalie 
#### Select relevant component teams: 
@codice/security 
#### Choose 2 committers to review/merge the PR. 
@clockard
@lessarderic
#### How should this be tested?
1. Start, and confirm that the `security-sts-propertyclaimshandler` bundle installs successfully. Install.
2. Follow the Documentation to update the `User Attribute File` location in the `File Based Claims Handler` configuration in the Admin Console. Confirm that the Handler is updated with the contents of the new file.
#### Any background context you want to provide?
The impl will be manually tested in https://github.com/codice/alliance/pull/453.
#### What are the relevant tickets?
[DDF-3466](https://codice.atlassian.net/browse/DDF-3466)
#### Checklist:

#### Notes on Review Process
For the general case, merging is gated on the following:

- [ ] Review by at least two contributors
- [ ] Review by at least two committers
- [ ] Successful CI
- [ ] Successful static analysis (SonarQube/Coverity/etc.)
- [ ] Heroing of change (details vary by ticket)

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
